scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -40,9 +36,9 @@
 ## normal_bam, normal_bam_index: BAM and index for the normal sample
 ##
 ## ** Primary resources ** (optional but strongly recommended)
-## pon, pon_index: optional panel of normals in VCF format containing probable technical artifacts (false positves)
```

Comment:
I think these index files are needed for the regular Mutect2.wdl, right?

Reply:
Also we should use either the `_idx` suffix or `_index`, not both

Reply:
done x 2